ios/ReactTestApp.xcodeproj/project.pbxproj

```diff
@@ -223,8 +223,8 @@
 		19ECD0CA232ED425003D8557 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastSwiftUpdateCheck = 1250;
-				LastUpgradeCheck = 1250;
+				LastSwiftUpdateCheck = 1300;
+				LastUpgradeCheck = 1300;
```

Comment:
probably just a nit, but might be worth waiting for this change to land for when Xcode 13.0 comes out? Or write a note somewhere about the fact that we're already using Xcode 13 here?

Or just do nothing, I guess it's mostly just you using this repo anyway 😅

Reply:
These two lines don't really do anything other than tell Xcode that we've done a code base "upgrade". There were no changes between 12 and 13 in RNTA, so these have no other effect than not prompting users to upgrade when they open a project with Xcode 13.